slave-base/Dockerfile.rhel7

```diff
@@ -17,7 +17,7 @@ ENV HOME=/home/jenkins
 USER root
 # Install headless Java
 RUN yum-config-manager --disable epel >/dev/null || : && \
-    INSTALL_PKGS="atomic-openshift-clients bc gettext git java-1.8.0-openjdk-headless java-1.8.0-openjdk-headless.i686 lsof rsync tar unzip which zip" && \
+    INSTALL_PKGS="bc gettext git java-1.8.0-openjdk-headless java-1.8.0-openjdk-headless.i686 lsof rsync tar unzip which zip" && \
```

Comment:
That's not a change in this pr, but we need java in the base to run the jenkins slave client. 

You may have seen in the maven image that i'm removing logic that previously removed(not added) the 1.7 jdk. That logic was needed before because the maven rpms incorrectly pulled in 1.7 when we were already installing 1.8 explicitly. That rpm dependency has been fixed, so we don't need that cleanup logic anymore.